Formula/zeromq@4.1.rb

```diff
@@ -0,0 +1,63 @@
+class ZeromqAT41 < Formula
+  desc "High-performance, asynchronous messaging library"
+  homepage "http://www.zeromq.org/"
+  url "https://github.com/zeromq/zeromq4-1/releases/download/v4.1.6/zeromq-4.1.6.tar.gz"
+  sha256 "02ebf60a43011e770799336365bcbce2eb85569e9b5f52aa0d8cc04672438a0a"
+
+  bottle do
+    cellar :any
+    sha256 "7d2de85d31cb632d0bb9aa94292c0023e2dd9b94849cf419124cd48540aaa4f8" => :sierra
+    sha256 "8677f4e36376d84752b00a2c9801c1fe8519ee9db7b6df051c52d203f1e9575e" => :el_capitan
+    sha256 "d987614f7d62306b6a2c9f51133f5e393678d25f5c8705821b40aebe956347b2" => :yosemite
+  end
+
+  keg_only :versioned_formula
+
+  option "with-libpgm", "Build with PGM extension"
+  option "with-norm", "Build with NORM extension"
+
+  deprecated_option "with-pgm" => "with-libpgm"
```

Comment:
A new formulae shouldn't have, or need, `deprecated_option` mappings as their purpose is to provide backwards compatibility with prior installations of that same formula.